packages/webapp/src/pages/index.tsx

```diff
@@ -12,6 +12,28 @@ import {
     Text,
 } from "_app";
 
+interface CommunityStatusProps {
+    memberStats: any
+}
+
+const CommunityStatsCard = ({memberStats}: CommunityStatusProps) => {
+    return memberStats && (
+        <Card className="flex flex-col justify-center items-center h-full space-y-8 text-md lg:text-xl">
+            <Heading size={2} className="mb-2">
+                Community Stats
+            </Heading>
+            <Link href="/members" className="font-medium">
+                {memberStats.active_members} active member
+                {memberStats.active_members !== 1 && "s"}
+            </Link>
+            <Link href="/induction/pending-invitations" className="font-medium">
+                {memberStats.pending_members} pending
+                    invitation
+                {memberStats.pending_members !== 1 && "s"}
+            </Link>
+        </Card>
+)}
+
```

Comment:
Let's move this component down below the main component. Sparky and I have agreed that the first component in a file should be the main one getting exported.